qiskit/algorithms/time_evolvers/trotterization/__init__.py

```diff
@@ -0,0 +1,19 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""This package contains Trotterization-based Quantum Real Time Evolution algorithm.
+It is compliant with the new Quantum Time Evolution Framework and makes use of
+:class:`qiskit.synthesis.evolution.ProductFormula` and
+:class:`~qiskit.circuit.library.PauliEvolutionGate` implementations."""
+
+from qiskit.algorithms.time_evolvers.trotterization.trotter_qrte import TrotterQRTE
+
+__all__ = ["TrotterQRTE"]
```

Comment:
I am wondering how to have this in the API ref. since it needs to be included in docs. The evolver interfaces we added Time, like time was added to evolvers pkg name, so they were different and the interfaces are listed in the algorithms.\_\_init\_\_.py file. The other algos that were never released can be imported/listed there too - PVQD and VarQITE etc. So how to do it for this... find a different name `QRTETrotter`,`QTrotterRTE`... or simply add `time_evolvers` link in algorithms and document this here in this init file. I recall the variational aspects added a pkg link too so maybe that works.